packages/trading/src/resolveSlippageSuggestion.ts

```diff
@@ -48,8 +49,15 @@ export async function resolveSlippageSuggestion(
       buyAmount: amountsAndCosts.afterSlippage.buyAmount,
     })
 
+    const suggestedSlippageBps = suggestedSlippage.slippageBps
+
     return {
-      slippageBps: Math.max(suggestedSlippage.slippageBps ?? 0, defaultSuggestion),
+      slippageBps: suggestedSlippageBps
```

Comment:
I intentionally cast `suggestedSlippageBps` to a boolean here to avoid zero values used as `volumeMultiplierPercent `